resources/istio/charts/kyma/templates/label-job.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kyma-ns-label
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyma-ns-label
+rules:
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["get", "list", "watch", "patch"]
```

Comment:
Please limit the permissions to namespace it should work on it (use names from values file)